extensions/sql-database-projects/src/projectProvider/projectProvider.ts

```diff
@@ -24,6 +24,16 @@ export class SqlDatabaseProjectProvider implements dataworkspace.IProjectProvide
 		return provider;
 	}
 
+	/**
+	 * Callback method when a project has been removed from the workspace view
+	 * @param projectFile The Uri of the project file
+	 */
+	RemoveProject(projectFile: vscode.Uri): Promise<void> {
+		// No resource release needed
+		console.log(`project file unloaded: ${projectFile.fsPath}`);
```

Comment:
remove console.log

Reply:
LOL, I added to avoid the unused variable error before I figure out how to bypass that check by adding comment.